lib/mixins/property-effects.js

```diff
@@ -2780,23 +2808,22 @@ export const PropertyEffects = dedupingMixin(superClass => {
     _removeBoundDom(dom) {
       // Unlink template info
       let templateInfo = dom.templateInfo;
-      if (templateInfo.previousTemplateInfo) {
-        templateInfo.previousTemplateInfo.nextTemplateInfo =
-          templateInfo.nextTemplateInfo;
-      }
-      if (templateInfo.nextTemplateInfo) {
-        templateInfo.nextTemplateInfo.previousTemplateInfo =
-          templateInfo.previousTemplateInfo;
+      const {previousSibling, nextSibling, parent} = templateInfo;
```

Comment:
Let's add a comment about how it's important that the parent is not unlinked.

Reply:
Done.